src/powershell/Public/Estimate-ProcessingTime.ps1

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+    .SYNOPSIS
+    This script is used to test the processing time of the ConvertTo-FinOpsSchema function. It is used inside the ConvertTo-FinOpsSchema.Tests.ps1 script. It is not meant to be run independently.
+
+    .DESCRIPTION
+    This script is used to test the ConvertTo-FinOpsSchema function.
+    You can specify the number of rows and columns to generate, as well as the column mapping.
+    It creates a CSV file based on the input of the user and calls the function with the specified parameters.
+    The function will create a new CSV file with the specified column mapping and export all columns or only the columns specified in the column mapping.
+    The script will then delete the input and output files.
+
+    .PARAMETER rows
+    Specifies the number of rows to use for the test.
+
+    .PARAMETER columns
+    Specifies the number of columns to use for the test.
+
+    .PARAMETER columnMapping
+    Specifies the column mapping to use for the test.
+
+    .PARAMETER ExportAllColumns
+    Specifies whether to export all columns or only the columns specified in the column mapping.
+
+#>
+
+
+<# Parameter Block #>
+[CmdletBinding()]
+param(
+    [int]
+    $rows,
+
+    [int]
+    $columns,
+
+    [hashtable]
+    $columnMapping,
+
+    [bool]
+    $ExportAllColumns = $true
+)
+    
+# Define the directory, input, and output file paths
+$directory = "..\Public"
```

Comment:
All the code here should be inside of a function. We can't add it outside of a function due to how the module gets built.

Reply:
This script will be replaced y Write-Progress